src/kili/queries/asset/helpers.py

```diff
@@ -73,30 +107,75 @@ def download_file(url: str, external_id: str, local_dir_path: Path) -> Path:
 
 def assert_required_fields_existence(asset: dict) -> None:
     """Get a list of an asset field from a generator of kili assets"""
-    required_fields = ["content", "externalId"]
+    required_fields = ["content", "externalId", "jsonContent", "project.inputType"]
```

Comment:
jsonContent and project.inputTypes are not in the default fields when querying assets.
This change would be breaking: a client wanting to download classic assets (with just content) would see his script fail because here you refuse to download his assets because he didn't ask to query the field jsonContent.
I think that we can't force people whi want to query "classic" assets to query the jsonContent field

Reply:
I added a warning if `jsonContent ` is not requested